packages/faceted-search/src/components/AddFacetPopover/AddFacetPopoverRow/AddFacetPopoverRowButton/AddFacetPopoverRowButton.scss

```diff
@@ -0,0 +1,30 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.tc-add-facet-popover-row-button {
+	font: tokens.$coral-paragraph-m-bold;
+	color: tokens.$coral-color-accent-text;
+	background-color: tokens.$coral-color-accent-text-weak;
+	border: none;
+	min-height: tokens.$coral-sizing-m;
+	padding: tokens.$coral-spacing-xxs tokens.$coral-spacing-m;
```

Comment:
```suggestion
	padding: tokens.$coral-spacing-xxs tokens.$coral-spacing-m;
	cursor: pointer;
	border-radius: 0px;
	...
```
If your app loses bootstrap (as it hopefully will someday) you'll need those resets

Reply:
Yes you are right but for now this component isn't even the long term way to implement our needs (https://github.com/Talend/ui/pull/4152#discussion_r914612007) so I prefer to keep it as simple as possible